pom.xml

```diff
@@ -223,7 +223,8 @@
             <groupId>com.worldcretornica</groupId>
             <artifactId>PlotMe-Core</artifactId>
             <version>0.17-SNAPSHOT</version>
-            <scope>provided</scope>
+            <scope>system</scope>
+            <systemPath>${basedir}/src/main/resources/PlotMe-Core.jar</systemPath>
```

Comment:
In a perfect world, one of us would figure out how to Maven-ize all the plugins and we'd use our own personal Depenizen repo or something instead of keeping them in the resources folder :cry: